lib/util/stream_utils.js

```diff
@@ -1578,6 +1594,16 @@ shaka.util.StreamUtils = class {
 };
 
 
+/**
+ * A cache of results from mediaCapabilities.decodingInfo, indexed by the
+ * (stringified) decodingConfig.
+ *
+ * @type {Object.<(!string), (!MediaCapabilitiesDecodingInfo)>}
+ * @export
```

Comment:
Why is this exported?  I don't think we want this exposed to the application and subject to backward compatibility guarantees.

If you just need it for unit testing, use `@suppress` in a test helper instead.